CMakeLists.txt

```diff
@@ -479,8 +479,11 @@ endif()
 
 option(AVIF_BUILD_APPS "Build avif apps." OFF)
 option(AVIF_BUILD_TESTS "Build avif tests." OFF)
+option(AVIF_BUILD_GTESTS
+       "Build avif C++ tests depending on GoogleTest. Requires GoogleTest to be installed and AVIF_BUILD_TESTS to be ON." OFF
```

Comment:
Nit: change "installed" to "present" or "available"?

If we use ext/googletest, then GoogleTest doesn't need to be installed on the machine.